template-deliverables.md

```diff
@@ -0,0 +1,145 @@
+# Template deliverables
+
+This document explains the different documentation deliverables that template authors are required to write as part of each template set. First, read our Template Contributing Guide to understand the larger context for these deliverables.
+
+## Table of contents
+
+- [Overview](#overview)
+- [Raw template](#raw-template)
+- [Template guide](#template-guide)
+- [Before you start guide (optional)](#before-you-start-guide-optional)

```

Comment:
I have some issues with calling this "Before you start guide", as well as more generally about what the purpose of this file is. More detail in previous comments.

Reply:
See my reply to your previous comments.

Reply:
I've updated this section based on the consensus we reached at the US general meeting last week: We discussed this at length at last week's general meeting and the group consensus there was to call this the "deep dive" document. It basically contains miscellaneous information or guidance that is too comprehensive for the "guide" document. Based on whatever the each unique template needs, this document could include research, deep theories, pre-writing information, and/or any other material that users need to know when implementing this topic. Template writers are encouraged to point/link readers to that document in their guide to encourage users to read it in more depth.